packages/pilot/src/containers/AddRecipient/ConfigurationStep/Anticipation/index.js

```diff
@@ -39,17 +40,26 @@ const anticipationModelOptions = ({
     value: 'automatic_dx',
   }
 
+  const custom = {
+    name: 'Customizado',
```

Comment:
```suggestion
    name: t('pages.add_recipient.custom_anticipation'),
```


esse name deve vir de uma tradução assim como é feito nos demais items desse arquivo

Reply:
Notei que já até existe uma translation para `Customizado`, bastando utiliza-la:

`t('pages.add_recipient.custom_anticipation')`

Reply:
Feito